src/Stratis.Bitcoin.Features.BlockStore/StoreSettings.cs

```diff
@@ -31,7 +27,7 @@ public class StoreSettings
         public bool PruningEnabled { get; set; }
 
         /// <summary>The maximum amount of blocks the cache can contain.</summary>
-        public int MaxCacheBlocksCount { get; set; }
+        public int MaxCacheSize { get; set; }
```

Comment:
Now it's not count but amount in MB 
Please update the comment and variable name